### ISSUE TITLE
Parsing Error

### ISSUE THREAD

**Issue by homj** [bpodgursky/jbool_expressions#3]
Hey Ben,

I just faced some issues when parsing a Expression from a String.
All Expressions are of the form "Expression<String>", but I usually put in numbers, formatted as Strings.
So there are two errors when parsing the Expressions from a String:

1) You can't parse sole variables, like: `0`

2) When trying to parse this Expression: `(0 & 1)` I get the errors:

```
W/System.err﹕ line 1:1 no viable alternative at character '0'
W/System.err﹕ line 1:3 no viable alternative at input '&'
W/System.err﹕ line 1:5 no viable alternative at character '1'
W/System.err﹕ line 1:6 no viable alternative at input ')'
```

I checked all the inputs and the Strings are formatted correctly.

So I think the parser just can't handle the numbers properly?
What would you suggest to use instead?

Thanks in advance and keep up the great work!

Johannes

**Comment by peplamb** [comment]
How do I add this to android studio?
I tried the following in build.gradle,

dependencies {
    compile 'com.bpodgursky.jbool_expressions:1.8'
}

but it doesn't work

**Comment by bpodgursky** [comment]
1.8 is not a released version.  The last released version is 1.7, try that.  You can see what versions are available searching the maven central repos;

http://search.maven.org/#browse%7C-308048032

**Comment by peplamb** [comment]
Great! thank you!
